devtools/generate_batch_config.py

```diff
@@ -55,7 +55,7 @@ def to_config(
                     ],
                     "computeResource": {
                         "cpuMilli": 8000,
-                        "memoryMib": 32 * 1024,
+                        "memoryMib": 64 * 1024,
```

Comment:
We saw that we're actually able to use more than 32 * 1024 MiB of memory:

https://console.cloud.google.com/monitoring/metrics-explorer;duration=P2D?referrer=search&project=catalyst-cooperative-pudl&pageState=%7B%22xyChart%22:%7B%22constantLines%22:%5B%5D,%22dataSets%22:%5B%7B%22plotType%22:%22LINE%22,%22targetAxis%22:%22Y1%22,%22timeSeriesFilter%22:%7B%22aggregations%22:%5B%7B%22crossSeriesReducer%22:%22REDUCE_SUM%22,%22groupByFields%22:%5B%22metric.label.%5C%22instance_name%5C%22%22%5D,%22perSeriesAligner%22:%22ALIGN_MEAN%22%7D%5D,%22apiSource%22:%22DEFAULT_CLOUD%22,%22crossSeriesReducer%22:%22REDUCE_SUM%22,%22filter%22:%22metric.type%3D%5C%22compute.googleapis.com%2Finstance%2Fmemory%2Fballoon%2Fram_used%5C%22%20resource.type%3D%5C%22gce_instance%5C%22%22,%22groupByFields%22:%5B%22metric.label.%5C%22instance_name%5C%22%22%5D,%22minAlignmentPeriod%22:%2260s%22,%22perSeriesAligner%22:%22ALIGN_MEAN%22%7D%7D%5D,%22options%22:%7B%22mode%22:%22COLOR%22%7D,%22y1Axis%22:%7B%22label%22:%22%22,%22scale%22:%22LINEAR%22%7D%7D%7D

Which I think means that we're getting each job allocated to a single `e2-highmem-8` instance, which has 64GB RAM. Because 32 * 1024 MiB > 32GB.

Since instance memory size appears to be in GB, not GiB, we want `61035` here to not bump us over into some even larger class of compute instance.

<img width="671" alt="image" src="https://github.com/catalyst-cooperative/pudl/assets/2495794/78373c06-56f1-4d42-818f-08b47c3d5c33">

Reply:
Ahhhhh, I see. I thought it had automatically bumped us to a bigger instance b/c it saw we were using more memory which seemed a little scary actually. This makes more sense. It just picks the smallest instance that'll provide the specified resources.  I'll bump this to 60 * 1024 just so it's more reflective of our actual usage.  Also this seems like a tricky way to get everybody to use larger instances than they actually need.